.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
   - docker pull camptocamp/geomapfish-build-dev-travis:${TRAVIS_BUILD_NUMBER}
   - docker pull camptocamp/geomapfish-commons-travis:${TRAVIS_BUILD_NUMBER}
   - docker pull camptocamp/geomapfish-build-travis:${TRAVIS_BUILD_NUMBER}
-  - docker tag camptocamp/geomapfish-build-dev-travis:${TRAVIS_BUILD_NUMBER}
+  - docker tag camptocamp/geomapfish-build-dev-travis:${TRAVIS_BUILD_NUMBER} camptocamp/geomapfish-build-dev:${MAJOR_VERSION}
       camptocamp/geomapfish-build-dev:${MAJOR_VERSION}
```

Comment:
Why adding twice the same tag? Is it even supported?

Reply:
camptocamp/geomapfish-build-dev-travis:${TRAVIS_BUILD_NUMBER}
!=
camptocamp/geomapfish-build-dev:${MAJOR_VERSION}
...